aiohttp/_websocket/models.py

```diff
@@ -70,36 +77,42 @@ def json(
 
 class WSMessagePing(NamedTuple):
     data: bytes
+    size: int
     extra: Optional[str] = None
     type: Literal[WSMsgType.PING] = WSMsgType.PING
 
 
 class WSMessagePong(NamedTuple):
     data: bytes
+    size: int
     extra: Optional[str] = None
     type: Literal[WSMsgType.PONG] = WSMsgType.PONG
 
 
 class WSMessageClose(NamedTuple):
     data: int
+    size: int
     extra: Optional[str] = None
     type: Literal[WSMsgType.CLOSE] = WSMsgType.CLOSE
 
 
 class WSMessageClosing(NamedTuple):
     data: None = None
+    size: int = 0
```

Comment:
I'm assuming this size allows us to skip the checks for message types in order to get a little extra performance?

Reply:
Shouldn't affect performance, and it was for convenience of not having to write it out for the synthetic messages since they are always going to be 0